src/App.tsx

```diff
@@ -1,14 +1,62 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+
+function prepareVisibleTodos(todos: Todo[], query: string, filterBy: string) {
+  let visibleTodos: Todo[] = [...todos];
+
+  if (query !== '') {
```

Comment:
```suggestion
  if (query) {
```
You don't need to compare with an empty string
https://developer.mozilla.org/en-US/docs/Glossary/Truthy